styles/mainStyle.css

```diff
@@ -0,0 +1,891 @@
+    color: rgba(9, 30, 66, 1);
+  }
+
+  .works .line {
+    flex-grow: 1;
+    width: auto;
+    margin: 0;
+  }
+
+  .gridContainer {
+    padding: 0 10vw;
+    grid-auto-rows: minmax(500px, auto);
+  }
+
+  .featured {
+    flex-direction: row;
+    align-items: flex-start;
+  }
+
+  .featured img {
+    flex: 1.5 0.5;
+  }
+
+  .featured .info {
+    flex: 1 1;
+    margin-left: 24px;
+  }
+
+  .card h3 {
+    font-family: 'Crete Round', sans-serif;
+    font-weight: 400;
+  }
+
+  .featured h3 {
+    margin-top: 0;
+    font-size: 40px;
+  }
+
+  .card p {
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  .featured .interaction {
+    margin-top: 24px;
+  }
+
+  .other {
+    display: none;
+  }
+
+  .other-desktop {
+    display: flex;
+  }
+
+  /* Modal */
+  .modal {
+    margin: 2em 8em;
+  }
+
+  .modal-grid {
+    grid-template-rows: 50vh;
+    grid-template-columns: 1.5fr 1fr;
+    height: 50vh;
+    width: 100%;
+  }
+
+  .modal-info {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+  }
+
+  /* About */
+  #about {
+    background-color: rgba(38, 38, 38, 1);
+  }
+
+  .about {
+    margin-left: 40px;
+    background-image: url(../images/about-desktop.svg);
+    background-position: right 0%;
+  }
+
+  .about p {
+    width: 46%;
+  }
+
+  .about h1 {
+    font-weight: 400;
+    color: rgba(245, 88, 0, 1);
+  }
+
+  .knowledge {
+    align-items: baseline;
+  }
+
+  .knowledge-parent {
+    flex-direction: row;
+    justify-content: space-around;
+    width: 100%;
+    box-sizing: border-box;
+  }
+
+  .element p {
+    font-size: 22px;
+  }
+
+  .categories h2 {
+    font-weight: 700;
+    font-size: 32px;
+  }
+
+  /* Contact Form */
+  #contact {
+    padding: 0 7vw;
+  }
+
+  .contact p {
+    font-family: "Inter", sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    text-align: left;
+    line-height: 52px;
+    margin-right: 24px;
+    color: rgba(38, 38, 38, 1);
+    flex: 1 1;
+  }
+
+  .contact form {
+    flex: 1.5 1;
+    align-items: baseline;
+  }
+
+  .contact-parent {
+    flex-direction: row;
+    align-items: baseline;
+  }
+
+  .other .interaction {
+    padding: 17px;
+  }
+
+  /* ID Styling */
+  #presentation h2 {
+    color: rgb(255, 255, 255);
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 500;
+  }
+
+  #presentation p {
+    color: rgba(244, 245, 247, 1);
+  }
+}
+
+@media screen and (min-width: 672px) and (max-width: 1000px) {
+  #presentation {
+    background-size: 25%;
+    height: auto;
+  }
+
+  #presentation .content {
+    padding: 25vh 0;
+    width: 60vw;
+  }
+}
+
+@media screen and (min-width: 672px) and (max-width: 1230px) {
+  .featured {
+    grid-column: 1/span 2;
+  }
+}
+
+@media screen and (min-width: 1190px) {
+  .gridContainer {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .featured {
+    grid-column: 1/span 3;
+  }
+
+  #presentation {
+    background-size: auto;
+  }
+}
```

Comment:
[OPTIONAL] Good you added these media queries. But the following lines of codes will be required later on in the portfolio application development.
Implementing them now in this specific task is not a good idea, please try not to go beyond the task requirements.